### PR TITLE
docs: include return type in Interval component properties

### PR DESCRIPTION
**Pull request by NickCrews**
Fixes https://github.com/ibis-project/ibis/issues/8165

### CODE REVIEW

**Comment by cpcloud** [comment]
@NickCrews Thanks for the PR. Is there any chance that instead of this approach we can add these methods and properties individually to `_quarto.yml` (likely requiring `dynamic: true` for each of them)?

**Comment by NickCrews** [comment]
[This is relevant](https://github.com/machow/quartodoc/issues/284). Now trying to poke around with the
```python
class A:
    attr_a: int
    """Docstring."""
```
format

**Comment by NickCrews** [comment]
I think this is still a gradual improvement, even if it isn't ideal. I think it shouldn't be bad to reverse or improve this later if this functionality changes. At the least, it adds `__add__` documentation, which was missing.

File: docs/_quarto.yml [hunk 1]
```diff
@@ -428,7 +428,26 @@ quartodoc:
...
-            - IntervalValue
+            - name: IntervalValue
+              dynamic: true
+              members:
+                - to_unit
+                - negate
+                - __add__
+                - __sub__
+                - __mul__
+                - __floordiv__
```
Comment (cpcloud) [inline docs/_quarto.yml:439]:
```suggestion
```

We're probably going to get rid of these methods soon, they're extremely finicky and don't really work as advertised.

**Review by cpcloud — CHANGES_REQUESTED** [review]

**Review by cpcloud — APPROVED** [review]
Thanks!

**Pull request merged**